posix.mak

```diff
@@ -25,9 +25,10 @@
 ################################################################################
 # Configurable stuff, usually from the command line
 #
-# OS can be linux, win32, win32wine, osx, or freebsd. The system will be
+# OS can be linux, win32, win32wine, osx, freebsd, netbsd or dragonflybsd. The system will be
 # determined by using uname
 
+#QUIET:=@
```

Comment:
Any reason for this to be added?

Reply:
It was needed while debugging, accidentally committed. Retracted now.